packages/perseus/src/widgets/radio/__stories__/focus-ring.stories.jsx

```diff
@@ -1,38 +1,38 @@
 // @flow
 import * as React from "react";
 
+import * as styleConstants from "../../../styles/constants.js";
 import FocusRing from "../focus-ring.jsx";
 
-type StoryArgs = {||};
+type StoryArgs = {|
+    children: React.Node,
```

Comment:
It's kind of silly (though I think it could be useful in the future), but I wanted a default element to show off characteristics of the focus ring while allowing it to be overwritten:
<img width="487" alt="Screen Shot 2022-10-05 at 4 07 11 PM" src="https://user-images.githubusercontent.com/16308368/194163985-af5cc0b5-3977-4a1b-8d7b-2a95e36eacb7.png">
<img width="495" alt="Screen Shot 2022-10-05 at 4 07 05 PM" src="https://user-images.githubusercontent.com/16308368/194164004-81924dd1-fa8b-4f16-9dcb-5afb370094cb.png">

Reply:
I see.  I hadn't thought about changing children via controls.  It does seem a little silly.  I don't have a strong opinion about keeping/dropping it.